src/Chapter1.hs

```diff
@@ -490,7 +491,9 @@ Implement a function that returns the last digit of a given number.
   whether it works for you!
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+
+lastDigit :: Int -> Int
+lastDigit n = (`mod` 10) $ abs n
```

Comment:
You can avoid function application and parentheses if you use the function in the infix form:
```suggestion
lastDigit n = abs n `mod` 10
```